code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -333,8 +333,12 @@ var/bomb_set
 	yes_code = 0
 	safety = 1
 	update_icon()
-	for(var/mob/M in player_list)
-		M << 'sound/machines/Alarm.ogg'
+	if(prob(1))
+		for(var/mob/M in player_list)
```

Comment:
This means we can easily add other exceptions for other sounds but the initial sound is the original and we have less code to maintain